api/core/tools/provider/builtin/jina/jina.yaml

```diff
@@ -1,10 +1,10 @@
 identity:
   author: Dify
-  name: jina
+  name: Jina AI
```

Comment:
Please keep this name the same as the provider's path name, for this provider, it's `jina`.